blueprint/index.md

```diff
@@ -0,0 +1,311 @@
+---
+
+# Conversation model to SQL Blueprint  
+
+This Genesys Blueprint provides an example of how to design an SQL database for storing JSON data, specifically from the [POST 
+/api/v2/analytics/conversations/details/query](https://developer.genesys.cloud/api/rest/v2/analytics/#post-api-v2-analytics-conversations-details-query) endpoint. A project containing a dockerized postgres database and Node.js Typescript driver will be included as an example.
+
+![ERDiagram](images/erdiagram.png "ERDiagram")
+
+## Solution components
+* **Genesys Cloud** - A suite of Genesys cloud services for enterprise-grade communications, collaboration, and contact center management.
+* **PostgreSQL** - An open source object-relational database system.
+* **Docker** - A set of platform as a service products that use OS-level virtualization to deliver software in packages called containers.
+* **Node.js** - An open-source, cross-platform, back-end JavaScript runtime environment that runs on the V8 engine and executes JavaScript code outside a web browser.
+
+## Software Development Kit (SDK)
+* **Genesys Cloud Platform API SDK** - This SDK is used for sending requests to the conversations details endpoint to obtain data for writing into the database.
+
+## Requirements
+
+### Specialized knowledge
+* SQL - This Blueprint uses PostgreSQL but knowledge of any SQL flavour will work
+* Typescript or Javascript - The driver has been written in Typescript using the Node.js runtime environment
+* Docker - Docker Compose has been used to simplify the interface with the docker container but an elementary knowledge of Docker would be helpful
+* Genesys Cloud Platform API knowledge
+
+### Genesys Cloud account requirements
+This solution requires a Genesys Cloud license. For more information on licensing, see [Genesys Cloud Pricing](https://www.genesys.com/pricing "Opens the pricing article").
+
+Client Credentials are used to authenticate with the Genesys Cloud API. The following scopes are required for making the call to [POST 
+  /api/v2/analytics/conversations/details/query](https://developer.genesys.cloud/api/rest/v2/analytics/#post-api-v2-analytics-conversations-details-query)
+* analytics
+* analytics:readonly
+
+## Database design
+
+The challenge in this task is to flatten a JSON structure to allow it to be written to an SQL database. The general approach has been to extract nested objects into their own tables containing a primary key and a foreign key referencing the primary key of the object above it. Where a primary key wasn't readily available for the segments and metrics tables, they were created by combining the foreign key with 2 other values in order to make it unique.
+Where structures are an array of strings, they have been stored in the database as a single string delimited by a comma. `divisionIds` and `requestedRoutings` are examples of this.
+
+The tables created to store the conversation data are as follows:
+* conversations - PK: conversation_id
```

Comment:
I'd suggest putting all of the column names in single backticks since they're literal values. There's also some independent underscores in some of the bullets, be sure those render the way you want them to. They might be making it italic.